vtm/src/org/oscim/tiling/source/mapfile/MapDatabase.java

```diff
@@ -963,12 +994,11 @@ private boolean processWays(QueryParameters queryParameters, ITileDataSink mapDa
                 }
 
                 if (labelPosition != null && wayDataBlock == 0)
-                    e.setLabelPosition(e.points[0] + labelPosition[0], e.points[1] + labelPosition[1]);
+                    e.setLabelPosition(mTileProjection.projectLon(labelPosition[1]),
+                            mTileProjection.projectLat(labelPosition[0]));
                 else
                     e.labelPosition = null;
 
-                mTileProjection.project(e);
```

Comment:
The `project` method performs several projections, are they all covered?
Or can improve the lines in PR and use it for all other data, like points, etc.

Reply:
In processWays(...)
mTileProjection.project(e) is the only usage of function project.

**Before:**
Readfile -> iterate through all points -> store in MapElement **e** -> iterate all points of **e** -> project point in **e**
**Now:**
Readfile -> iterate through all points -> store projected point in MapElement **e**

Points where already projected correctly while reading in the mapfile using:
```suggestion
mTileProjection.projectPoint(latitude, longitude, e);
```